resources/views/components/section-header/back-link.blade.php

```diff
@@ -0,0 +1,26 @@
+@props([
+    'title',
+    'extraLabel' => null,
+    'href' => null,
+    'backLabel' => 'Back',
```

Comment:
```suggestion
    'href',
    'backLabel',
```

i would prefer if these are not optional since you will need an `href` for the component to work properly and there isn't really a "default label" because we use different labels everywhere